paddle/fluid/operators/mkldnn/split_mkldnn_op.cc

```diff
@@ -95,18 +96,14 @@ class SplitMKLDNNKernel : public framework::OpKernel<T> {
 
     auto x_vec_dims = phi::vectorize(x_dims);
 
-    dnnl::memory::data_type x_type =
-        framework::ToMKLDNNDataType(framework::TransToProtoVarType(x->dtype()));
+    dnnl::memory::data_type x_type = phi::funcs::ToMKLDNNDataType(x->dtype());
 
     auto& astream = platform::MKLDNNDeviceContext::tls().get_stream();
 
     std::vector<int64_t> offset(x_vec_dims.size(), 0);
 
-    platform::ReorderMKLDNNHandler reorder_handler(
-        x_vec_dims,
-        framework::TransToProtoVarType(x->dtype()),
-        x_type,
-        onednn_engine);
+    phi::funcs::ReorderMKLDNNHandler reorder_handler(
```

Comment:
maybe ReorderMKLDNNHandler put in `phi/backends/onednn` is better? It is a onednn-library related class, so it is put in `platform` dir originally

Reply:
done in #45659